PR: https://github.com/odoo/odoo/pull/86332

From: 24f9af3fd088c0733bb1559373955e90e23efd2c
From: Victor Feyens
Date: 2022-04-27 05:51:23

Structural Changes: 3
Total Changes: 6

[IMP] *: remove Trailing newlines (C0305)

Part-of: odoo/odoo#86332

================================= pseudo patch: =================================

--- a/addons/stock_landed_costs/models/__init__.py
+++ b/addons/stock_landed_costs/models/__init__.py
@@ -8,4 +8,3 @@ from . import res_config_settings
 from . import stock_landed_cost
 from . import account_move
 from . import stock_valuation_layer
-

--- a/addons/stock_landed_costs/models/res_company.py
+++ b/addons/stock_landed_costs/models/res_company.py
@@ -8,4 +8,3 @@ class ResCompany(models.Model):
     _inherit = 'res.company'
 
     lc_journal_id = fields.Many2one('account.journal')
-

--- a/addons/stock_landed_costs/models/res_config_settings.py
+++ b/addons/stock_landed_costs/models/res_config_settings.py
@@ -8,4 +8,3 @@ class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
     lc_journal_id = fields.Many2one('account.journal', string='Default Journal', related='company_id.lc_journal_id', readonly=False)
-

--- a/addons/stock_landed_costs/models/stock_valuation_layer.py
+++ b/addons/stock_landed_costs/models/stock_valuation_layer.py
@@ -5,9 +5,6 @@ from odoo import fields, models
 
 
 class StockValuationLayer(models.Model):
-    """Stock Valuation Layer"""
-
     _inherit = 'stock.valuation.layer'
 
     stock_landed_cost_id = fields.Many2one('stock.landed.cost', 'Landed Cost')
-
